keyboards/owlab/jelly_evolv/solder/readme.md

```diff
@@ -0,0 +1,21 @@
+# Jelly Evolv - soldered PCB
+
+![jelly evolv](https://i.imgur.com/Kk49RUVh.jpg)
+
+A 75% made by Owlab.
```

Comment:
If these match the 75% layouts in the community layouts, then community layout support would be great. 

https://github.com/qmk/qmk_firmware/tree/master/layouts/default

https://docs.qmk.fm/#/feature_layouts

Reply:
Hi, our layout is slightly different from the community 75% layout so I am afraid we'll stay with the current one. Thanks anyway.